packages/react-native-client/android/build.gradle

```diff
@@ -35,7 +35,7 @@ android {
   namespace "com.swmansion.reactnativeclient"
   defaultConfig {
     versionCode 1
-    versionName "0.1.3"
+    versionName "0.1.4"
```

Comment:
NIT: WDYT about changing it to use `version` variable from the top of file? (I think Fishjam Cloud has it configured in that way)